src/frontend/displaywindow/cdisplaywindow.cpp

```diff
@@ -275,6 +275,8 @@ CDisplayWindow::CDisplayWindow(BtModuleList const & modules,
             initAddAction(QStringLiteral("copySelectedText"),
                           m_displayWidget,
                           &BtModelViewReadDisplay::copySelectedText);
+    copySelectedTextAction.setEnabled(
+        displayWidget()->qmlInterface()->selection().has_value());
```

Comment:
Please use `m_displayWidget` instead of `displayWidget()`. Otherwise, looks good to me. :)

Reply:
Done.